packages/fxa-content-server/grunttasks/l10n-extract.js

```diff
@@ -19,7 +19,7 @@ var messagesOutputPath = path.join(
   'LC_MESSAGES'
 );
 var babelCmd =
-  'npx babel --plugins=@babel/plugin-syntax-dynamic-import,dynamic-import-webpack,@babel/plugin-proposal-class-properties --presets @babel/preset-react,@babel/env app/scripts --out-dir .es5';
+  'npx babel --plugins=@babel/plugin-syntax-dynamic-import,dynamic-import-webpack,@babel/plugin-proposal-class-properties --presets @babel/preset-react,@babel/env,@babel/preset-typescript app/scripts --out-dir .es5';
```

Comment:
I also added @babel/preset-typescript because we use it when doing normal compilations.